examples/tests/valid/container_types.w

```diff
@@ -59,10 +59,20 @@ assert(m7.size() == 3);
 assert(m7.get("b") == 2);
 assert(m7.has("b"));
 assert(m4.has("boom") == false);
-let m8 = {"a": "a1", b: "b1", c: "c1"};
+let m8 = {"a": "a1", "b": "b1", "c": "c1"};
 assert(m8.keys().at(0) == "a");
 assert(m8.keys().at(1) == "b");
 assert(m8.keys().at(2) == "c");
+assert(m8.values().at(0) == "a1");
+assert(m8.values().at(1) == "b1");
+assert(m8.values().at(2) == "c1");
+let m9 = MutMap<str>{"a": "a1", "b": "b1", "c": "c1"};
+assert(m9.keys().at(0) == "a");
+assert(m9.keys().at(1) == "b");
+assert(m9.keys().at(2) == "c");
+assert(m9.values().at(0) == "a1");
+assert(m9.values().at(1) == "b1");
+assert(m9.values().at(2) == "c1");
```

Comment:
Can we also add a few lines to show it's possible to iterate over the return value? For example:

```
for val in m9.keys() {
  assert(!val.endsWith("1"));
}
for val in m9.values() {
  assert(val.endsWith("1"));
}
```

Reply:
Sure, good hint! ✅